static/styles/draft_results.css

```diff
@@ -0,0 +1,29 @@
+table {
+  border-collapse: collapse;
+  width: 100%;
+}
+
+table {
```

Comment:
are one of these intended to be .table? (that said see below, wouldnt put css directly on the tags.

if we do want something of that global of a nature we probably want to put it into our base css and have that formatting apply to all tables